### DISCUSSION TITLE
steamguard or steamguard-cli?

### DISCUSSION THREAD

**Discussion by rivenirvana** [dyc3/steamguard-cli#199] [Q&A] [answered]
Why does the readme say `steamguard` when the crate installs the executable `steamguard-cli`?

**Comment by dyc3** [comment]
Yeah this is kind of annoying. `steamguard` is the canonical way to invoke steamguard-cli from your terminal... or at least that's how I _want_ it to work. There's limitations with cargo that prevent me from changing the name of the emitted binary right now: https://github.com/dyc3/steamguard-cli/blob/7950e43f706d9337df8061f7578fef71d8eb38d4/Cargo.toml#L22-L24

**Reply by dyc3** [reply, marked as answer]
This has now been fixed with a kinda janky workaround for now. `steamguard-cli` should now always be built as `steamguard`

**Reply by rivenirvana** [reply]
Thanks! Hope the appropriate fix from cargo lands soon.
